### PR TITLE
mdd-252983

### PR DESCRIPTION
**Pull request by mddifilippo89**
mdd-252983

### CODE REVIEW

File: en/components/excel_library.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
----
+﻿---
 title: Excel Library Component - Native Angular | Ignite UI for Angular
 _description: The Ignite UI for Excel Library component ... TODO.
 _keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Excel Library, Angular Excel Library Example, Angular Excel Library Component, Angular Excel Engine
 ---
 ## Excel Library
 
-The Excel Library is ... TODO.
+The Infragistics Excel Engine allows you to work with spreadsheet data using familiar Microsoft® Excel® spreadsheet objects like Workbooks, Worksheets, Cells, Formulas and many more. The Infragistics Excel Engine makes it easy for you to represent the data of your application in an Excel spreadsheet as well as transfer data from Excel into your application.
```
Comment (HUSSAR-mtrela) [inline en/components/excel_library.md]: we should keep it consistent and use either "Excel Library" or "Excel Engine" in all docs and TOC

File: en/components/excel_library.md [hunk 2]
```diff
@@ -43,37 +40,45 @@ import { IgxExcelModule } from "igniteui-angular-excel/ES5/igx-excel-module";
...
-
-```typescript
-var data = ... TODO
-
-```
+### Load and Save Workbooks
+Now that the Excel Library module is imported, next step is to load a workbook.
 
-The following code demonstrates how to bind the `Workbook` object to the above data.
+In order to load and save `Workbook` object, you can utilize to load and save methods Excel. The following code is a class that exposes various methods such as load and save.
```
Comment (pamelabrasil) [inline en/components/excel_library.md]: utilize the load and save

File: en/components/toc.yml [hunk 3]
```diff
@@ -154,24 +154,24 @@
...
+    # - name: Using Tables
+    #   href: excel_library_using_tables.md
+    #   new: true
+    # - name: Using Workbooks
+    #   href: excel_library_using_workbooks.md
+    #   new: true
+    # - name: Using Worksheets
+    #   href: excel_library_using_worksheets.md
+    #   new: true
     - name: Working with Charts
```
Comment (HUSSAR-mtrela) [inline en/components/toc.yml]: suggest using these titles for topics:
- Workbook Operations
- Worksheet Operations
- Working with Cells
- Working with Tables

File: en/components/excel_library.md [hunk 4]
```diff
@@ -47,33 +45,43 @@ It's also possible to load a more minimal configuration of Excel Library whereby
...
 import { Workbook } from "igniteui-angular-excel/ES5/Workbook";
+import { WorkbookSaveOptions } from "igniteui-angular-excel/ES5/WorkbookSaveOptions";
 import { WorkbookFormat } from "igniteui-angular-excel/ES5/WorkbookFormat";
-import { Worksheet } from "igniteui-angular-excel/ES5/Worksheet";
-import { WorksheetTable } from "igniteui-angular-excel/ES5/WorksheetTable";
 
-TODO
+Workbook.load("ExcelWorkbook.xlsx", null, null);
+Workbook.save(workbook, "ExcelWorkbook.xlsx);
+
```
Comment (HUSSAR-mtrela) [inline en/components/excel_library.md]: are you sure we have these methods on workbook? If not then maybe we should just show:
var wb = new Workbook(format);
var ws2020 = wb.worksheets().add("Budget2020"); 
var ws2021 = wb.worksheets().add("Budget2021");

File: en/components/toc.yml (hunk 3 above)
Comment (mddifilippo89) [reply en/components/toc.yml]: ok, I'll fix this when we enable them back

**Comment by zdrawku** [comment]
@HUSSAR-mtrela and @pamelabrasil each topic that contains EN changes (new or updated) should follow the Controbuting guidance. For example, the changes above should be applied to the JP and KR md equivalents and status:pending-localization should be added to the PR. This is requirement that helps to our localization team NOT to miss any changes.

**Pull request merged**